packages/cypress/src/junit.ts

```diff
@@ -120,8 +124,11 @@ function appendReplayUrlsToFailureNodes(node: INode, replayUrls: string[]) {
         return;
       }
 
-      failure.children[0] +=
-        "\n\n View in Replay\n" + replayUrls.map(url => ` * ${url}`).join("\n");
+      const output = `${failure.children[0]}\n\nView in Replay\n${replayUrls
+        .map(url => ` * ${url}`)
+        .join("\n")}`;
+
+      failure.children[0] = escapeForXml(output);
```

Comment:
If we do string replacement on whole output, then won't it also replace ampersand in URL's queryparams? Like if url is app.replay.io/recording/some-id?test=1&somethingelse=1, it'll be converted to app.replay.io/recording/some-id?test=1&amp;somethingelse=1. 🤔

Reply:
It has to be encoded for XML output. When decoded, it'll revert to a valid URL.

Reply:
Also, sort of funny, your comment showed exactly what happens because it converted the entity sequence back to &